rep-2000.rst

```diff
@@ -547,6 +547,128 @@ Build System Support:
 - cmake
 - setuptools
 
+Foxy Fitzroy (May 2020 - May 2023)
+----------------------------------
+
+Targeted platforms:
+
++--------------+-----------------------+-------------------------+----------------------+--------------------+---------------+
+| Architecture | Ubuntu Focal (20.04)  | MacOS Catalina (10.15)* | Windows 10 (VS2019)  | Debian Buster (10) | OpenEmbedded  |
+|              |                       |                         |                      |                    | Thud (2.6) /  |
```

Comment:
@andre-rosa @herb-kuta-lge How about targeting a new version Open Embedded with the ROS 2 release in May 2020? Warrior 2.7, Zeus 3.0, or even a newer upcoming one?

Reply:
@dirk-thomas  < We can definitely consider upgrade of yocto for Foxy. We already added warrior to our roadmap. We will review zeus and decide if we want to consider zeus instead of warrior or add both.

Reply:
@dirk-thomas Here is our proposal for determining which OpenEmbedded release series are supported for a ROS distro.

Reply:
> Here is our proposal for determining which OpenEmbedded release series are supported for a ROS distro.

What do you mean with "here"?

Reply:
Oops: forgot the link: here = https://github.com/ros/meta-ros/issues/665

Reply:
@herb-kuta-lge Please propose a pull request targeting the branch of this PR to update it accordingly.

Reply:
Reading the current release policy for OpenEmbedded (https://github.com/ros/meta-ros/wiki/Policies#openembedded-release-series-support) it seems that it doesn't match the promised support time frame of ROS distros. Support for a ROS distro for OpenEmbedded could be ended before the date promised in this REP. Is that correct? If that is the case I thin this needs a *big* note that this is the case since otherwise users will expect the support dates mentioned in this document to apply to OpenEmbedded.

Reply:
> Please propose a pull request targeting the branch of this PR to update it accordingly.

@herb-kuta-lge Also please follow up on this previous comment.

Reply:
@dirk-thomas < Every (non-EOL) ROS distro will always supported under at least one stable OpenEmbedded release series; it's just which one will change throughout the lifetime of the ROS distro. This is why we have dropped "Thud" from the OpenEmbedded column titles and added "\*\*" to indicate that it's a "rolling distribution" in [PR 234](https://github.com/ros-infrastructure/rep/pull/234/commits/70a3873278b42509aee90143431bfe67aa439108). Would it be clearer if we spelled this out in the explanation of the "\*\*" indication?

Reply:
> Would it be clearer if we spelled this out in the explanation of the "**" indication?

Yes, same as https://github.com/ros-infrastructure/rep/pull/234#pullrequestreview-378063201

Reply:
I think this is the last thread to be resolved for the PR. Anything I can do to help move this forward?

We could also open a PR against the REP after it has merged.

Reply:
Applying the same changes as #234 should be sufficient.

Reply:
See f487cee